test/Microsoft.AspNetCore.Mvc.Core.Test/RequireHttpsAttributeTests.cs

```diff
@@ -141,6 +149,47 @@ public void HandleNonHttpsRequestExtensibility()
             Assert.Equal(StatusCodes.Status404NotFound, result.StatusCode);
         }
 
+        [Theory]
+        [InlineData("http://localhost", null, "https://localhost/")]
+        [InlineData("http://localhost:5000", null, "https://localhost/")]
+        [InlineData("http://[2001:db8:a0b:12f0::1]", null, "https://[2001:db8:a0b:12f0::1]/")]
+        [InlineData("http://[2001:db8:a0b:12f0::1]:5000", null, "https://[2001:db8:a0b:12f0::1]/")]
+        [InlineData("http://localhost:5000/path", null, "https://localhost/path")]
```

Comment:
Thanks @sebastienros Just FYI...you seem to have squashed your commits. This makes it difficult to find the differences(to see how the feedback was addressed) from earlier commit. You could squash the commits once you get a sign off.